### DISCUSSION TITLE
Поиск MeiliSearch

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#481] [Как это сделать?] [answered]
Ошибка при использовании поиска MeiliSearch

**Comment by ttm-alexeypolunin** [comment, marked as answer]
**Удалена таблица, по которой был сделан индекс:**

<img width="1555" alt="Снимок экрана 2023-12-04 в 18 27 19" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/12c5dfb7-5cd4-4389-95ad-f17558dbdeef">

**Вам нужно удалить ее из списка настроек и потом удалить все ее строки из индекса поиска:**

<img width="1567" alt="Снимок экрана 2023-12-04 в 18 33 03" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/879dfd7a-1e4b-4339-9e0b-0bf80d548f53">

**Reply by Makhmut2445** [reply]
Спасибо. Получилось
